docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
       - KAIJU_RAILS_SERVER_URL=http://proxy
     labels:
       - "traefik.backend=node"
-      - "traefik.frontend.rule=Path: /projects/{id}/workspaces/{id}/code, /projects/{id}/workspaces/{id}/preview, /projects/{id}/workspaces/{id}/preview_files/{id}, /projects/{id}/workspaces/{id}/code_files/src/{id}"
+      - "traefik.frontend.rule=PathPrefix: /projects/{id}/workspaces/{id}/preview, /projects/{id}/workspaces/{id}/preview_files, /projects/{id}/workspaces/{id}/code_files"
```

Comment:
The code path is now hitting the rails code generation, is this what you wanted?

Reply:
Yes, the node route returns a index of generated files. Not exactly the most presentable. We need to do some work on that route before we can present it to people face up.

Reply:
I see that my summary was miss-worded. let me correct that.